### ISSUE TITLE
Inadequate color contrast: admin dashboard `.actions` icons

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2305]
The `.actions` icons that appear on admin side do not have adequate contrast for UI components.
<img width="306" alt="Actions icons, which are styled in a light red / pinkish tone" src="https://github.com/user-attachments/assets/1c989030-9245-41eb-a1d2-ac69d8376296" />
They are styled with #A91919 at a 0.5 opacity (translates to ~ #D48D8E), which results in a 2.6:1 contrast. These should have at least a 3:1 contrast.

**Comment by aviannamiller** [comment]
The updated contrast looks good!

This change does not appear to be fully picking up in a few areas across core (and one module):

- On items, media, and item set advanced search pages: icons are not picking up on the new hover style, though they have good contrast otherwise.
- A site's Navigation configuration as well as the Hierarchy module (under Modules, not Site Hierarchies) employ `.jstree-icon` icons, which do not appear to pick up on the new styles at all.

**Comment by kimisgold** [comment]
Good catches. I've applied the new hover states to those areas.
